pkg/shim/util_unix.go

```diff
@@ -286,7 +290,12 @@ func cleanupSockets(ctx context.Context) {
 	}
 	if len(socketFlag) > 0 {
 		_ = RemoveSocket("unix://" + socketFlag)
-	} else if address, err := SocketAddress(ctx, addressFlag, id); err == nil {
+	} else if address, err := SocketAddress(ctx, addressFlag, id, false); err == nil {
+		_ = RemoveSocket(address)
+	}
+	if len(debugSocketFlag) > 0 {
+		_ = RemoveSocket("unix://" + debugSocketFlag)
+	} else if address, err := SocketAddress(ctx, addressFlag, id, true); err == nil {
```

Comment:
on this line, it should be `debugSocketFlag` right?

Reply:
This should be `addressFlag`. The idea is that if `debugSocketFlag` is specified, its value would be the socket address. otherwise the socket address should be derived from the combo of `addressFlag` and `id`.